src/quo2/components/inputs/input/view.cljs

```diff
@@ -55,16 +57,19 @@
 (def ^:private custom-props
   "Custom properties that must be removed from properties map passed to InputText."
   [:type :blur? :theme :error? :right-icon :left-icon :disabled? :small? :button
-   :label :char-limit :on-char-limit-reach :icon-name :multiline? :on-focus :on-blur])
+   :label :char-limit :on-char-limit-reach :icon-name :multiline? :on-focus :on-blur
+   :container-style])
 
 (defn- base-input
-  [{:keys [on-change-text on-char-limit-reach container-style weight]}]
+  [{:keys [on-change-text on-char-limit-reach weight]}]
   (let [status              (reagent/atom :default)
         internal-on-focus   #(reset! status :focus)
         internal-on-blur    #(reset! status :default)
         multiple-lines?     (reagent/atom false)
-        set-multiple-lines! #(let [height (oops/oget % "nativeEvent.contentSize.height")]
-                               (if (> height 57)
+        set-multiple-lines! #(let [height     (oops/oget % "nativeEvent.contentSize.height")
+                                   ;; In Android height comes with padding
+                                   min-height (if platform/android? 40 22)]
+                               (if (> height min-height)
```

Comment:
We are having many problems related to size in the text input multiline. They are being hard to handle to look exactly as designs.
Please take a look at:
https://github.com/facebook/react-native/issues/29702

Maybe when they fix their problems we are going to be able to easily match designs